api/proto/config/devcommon.proto

```diff
@@ -49,4 +49,14 @@ message ConfigItem {
 message Adapter {
   org.lfedge.eve.common.PhyIoType type = 1;
   string name = 2;	// Short hand name such as "com" from bundle
+  EthVF eth_vf = 3; // Only relevant for PHY_IO_TYPE_PHY_IO_NET_ETH_VF
 }
+
+// Information regarding Virtual Function (VF) customisation
+message EthVF {
+  uint32 index = 1;
+  string pci = 2;
```

Comment:
User has option to configure it manually, it's very unlikely, and it is "do it at your own risk" kind of situation

Reply:
Could you please point me to the place in the code where this is applied for a VF?

Reply:
You're right it's not implemented on our side, I did further investigation on how user can set up PCI address for the VF and found interesting solution for it from RedHat [here](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/virtualization_deployment_and_administration_guide/sect-pci_devices-pci_passthrough) section 16.2.4, citing below:

> Hard coding the PCI addresses of particular Virtual Functions (VFs) into a guest's configuration has two serious limitations:
> - The specified VF must be available any time the guest virtual machine is started. Therefore, the administrator must permanently assign each VF to a single guest virtual machine (or modify the configuration file for every guest virtual machine to specify a currently unused VF's PCI address each time every guest virtual machine is started).
> - If the guest virtual machine is moved to another host physical machine, that host physical machine must have exactly the same hardware in the same location on the PCI bus (or the guest virtual machine configuration must be modified prior to start).
> 
> It is possible to avoid both of these problems by creating a libvirt network with a device pool containing all the VFs of an SR-IOV device. Once that is done, configure the guest virtual machine to reference this network. Each time the guest is started, a single VF will be allocated from the pool and assigned to the guest virtual machine. When the guest virtual machine is stopped, the VF will be returned to the pool for use by another guest virtual machine.
> 

I think that for now we should remove PCI field from here and implement pool in next releases (if it'll be needed, which I'm having doubts)